### ISSUE TITLE
Connect - Country Profile Page -no search field

### ISSUE THREAD

**Issue by ChristopherChuckA** [UBT-Designers/Connect-Web-App#1]
@lulomunoz 

attached are the link created in invissionapp on the revised C Profile page. 

https://invis.io/NX8J4XA85

I added a divider between the localities image and country name/"other countries"dropdown menu. 
![image](https://cloud.githubusercontent.com/assets/22042976/18334049/f69c4f00-75a6-11e6-9eb2-d867feeec142.png)

I added it on purpose because if locals are too many we can add a scroll option in localities image and leave the country name and dropdown menu in fixed position. 

https://invis.io/ZH8J4YQ63

CC: @lekzen @kennethgaerlan

**Comment by ChristopherChuckA** [comment]
@lulomunoz 

btw i got the style of the scroll bar here: https://codecanyon.net/item/css3-scrollbar-styles/6122412

**Comment by lulomunoz** [comment]
@ChristopherChuckA 

This is totally right, I just sent you an email with all the localities list, please add more so it covers the empty space, you can add the scroll to show that is scrollable.

Thanks!

**Comment by ChristopherChuckA** [comment]
@lulomunoz 

here is the link for scroll sample in invissionapp: 

https://projects.invisionapp.com/share/ZH8J4YQ63#/screens
